resources/dicts/lead_ceremony_df.json

```diff
@@ -329,7 +1114,8 @@
       "tags": [],
       "lead_trait": [],
       "text": [
-            "r_c hails m_c by {PRONOUN/m_c/poss} new name, m_c_star, telling them that {PRONOUN/m_c/poss} old, pathetic life is no more. {PRONOUN/m_c/subject} {VERB/m_c/are/is} granted guardianship of c_n, and are told to use {PRONOUN/m_c/poss} new power however they wish. The group of shadowy cats yowls m_c_star's name in support. m_c_star wakes up feeling a new strength within {PRONOUN/m_c/poss} body and know that {PRONOUN/m_c/subject} {VERB/m_c/are/is} now ready to lead the Clan."
+            "r_c hails m_c by {PRONOUN/m_c/poss} new name, m_c_star, telling them that their old pathetic life is no more. {PRONOUN/m_c/subject/CAP} are granted guardianship of c_n, and are told to use their new power however they wish. The group of shadowy cats yowls m_c_star's name in support. m_c_star wakes up feeling a new strength within their body and know that they are now ready to lead the Clan."
```

Comment:
'their' needs a pronoun tag, 'are' needs a verb tag, last sentence is missing all pronoun tags